### DISCUSSION TITLE
使用pnpm安装vue无法引入@vue/runtime-core，但是使用yarn可以

### DISCUSSION THREAD

**Discussion by 379949990** [pnpm/pnpm#5786] [Q&A]
<img width="845" alt="image" src="https://user-images.githubusercontent.com/66375788/207203443-7af50540-87be-4858-8f8c-c1c2e2a063bb.png">
<img width="837" alt="image" src="https://user-images.githubusercontent.com/66375788/207203620-e930a6a1-57c4-4fe6-940b-cdb00d56df85.png">

- 这是一个来复现问题的极简的demo；
- 通过pnpm init并且仅安装了vue，但当使用pnpm安装时运行“node index”将会抛出错误：
UnhandledPromiseRejectionWarning: Error [ERR_MODULE_NOT_FOUND]: Cannot find package '@vue/runtime-core' imported from /Users/cong/Wuitu/_Temp/pnpm-demo/index.js
- 使用yarn则不会；
- 专注于@vue/runtime-core是因为unplugin-vue-components生成的components.d.ts中会自动生成引入@vue/runtime-core的代码，这使我的vue3项目始终出现错误
<img width="814" alt="image" src="https://user-images.githubusercontent.com/66375788/207204226-1f893fc9-6022-4ddc-a5bb-ad75ded08a4b.png">

- 寻求解决方案

**Comment by lvqq** [comment]
What's your pnpm version? It works fine in my machine under pnpm 7.18.2:

![image](https://user-images.githubusercontent.com/29809148/207210258-a6ed1ecf-13da-4b0c-94cc-d8d695fabe61.png)
